src/clusterfuzz/_internal/protos/uworker_msg.proto

```diff
@@ -89,7 +89,14 @@ message Input {
 }
 
 message SymbolizeTaskOutput {
- // TODO(metzman): Fill this out.
+  optional int64 build_fail_wait = 1;
```

Comment:
We don't need to pass this around. Just read this from the environment variable on the tworker.